src/elements/content-sidebar/SidebarContentInsights.scss

```diff
@@ -0,0 +1,12 @@
+.bcs-SidebarContentInsights {
+    .ContentAnalyticsErrorState {
+        svg {
+            margin-top: 25px;
+        }
+    }
+
+    .ContentAnalyticsErrorState-text,
+    .ContentAnalyticsErrorState-text--permission {
+        margin-bottom: 30px;
```

Comment:
Should we add padding to the container, instead?

Reply:
Good idea, fixing it